php-frontend/src/main/java/org/sonar/php/tree/symbols/NamespaceNameResolvingVisitor.java

```diff
@@ -70,7 +71,10 @@ public void visitUseStatement(UseStatementTree tree) {
     tree.clauses().forEach(useClauseTree -> {
       String alias = getAliasName(useClauseTree);
       SymbolQualifiedName originalName = getOriginalFullyQualifiedName(namespacePrefix, useClauseTree);
-      aliases.put(alias.toLowerCase(Locale.ROOT), originalName);
+      if (!aliasesPerNamespace.containsKey(currentNamespace)) {
+        aliasesPerNamespace.put(currentNamespace, new HashMap<>());
+      }
+      aliasesPerNamespace.get(currentNamespace).put(alias.toLowerCase(Locale.ROOT), originalName);
```

Comment:
I could imagine, that Optionals and flatMap could reduce the complexity

Reply:
I found this alternative :
`aliasesPerNamespace.computeIfAbsent(currentNamespace, k -> new HashMap<>()).put(alias.toLowerCase(Locale.ROOT), originalName);`